hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/ContainerOperationClient.java

```diff
@@ -449,11 +504,7 @@ public void closeContainer(long containerId)
   public long getContainerSize(long containerID) throws IOException {
     // TODO : Fix this, it currently returns the capacity
     // but not the current usage.
-    long size = getContainerSizeB();
-    if (size == -1) {
-      throw new IOException("Container size unknown!");
-    }
-    return size;
+    return containerSizeB;
```

Comment:
The parameter `containerID` is not used in this method. Do we still need to pass it?